test/testdrive/session.td

```diff
@@ -50,12 +50,14 @@ max_identifier_length               255                     "The maximum length
 max_kafka_connections               1000                    "The maximum number of Kafka connections in the region, across all schemas (Materialize)."
 max_materialized_views              100                     "The maximum number of materialized views in the region, across all schemas (Materialize)."
 max_mysql_connections               1000                    "The maximum number of MySQL connections in the region, across all schemas (Materialize)."
+max_network_policies                25                      "The maximum number of network policies in the region."
 max_objects_per_schema              1000                    "The maximum number of objects in a schema (Materialize)."
 max_postgres_connections            1000                    "The maximum number of PostgreSQL connections in the region, across all schemas (Materialize)."
 max_query_result_size               "1GB"                   "The maximum size in bytes for a single query's result (Materialize)."
 max_replicas_per_cluster            5                       "The maximum number of replicas of a single cluster (Materialize)."
 max_result_size                     "1GB"                   "The maximum size in bytes for an internal query result (Materialize)."
 max_roles                           1000                    "The maximum number of roles in the region (Materialize)."
+max_rules_per_network_policies      25                      "The maximum number of rules per network policies."
```

Comment:
Should this session parameter be called `max_rules_per_network_policy` (singular)?